Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java

```diff
@@ -231,10 +232,11 @@ public void run() {
 
         try {
             progressMonitor.setProgressText(Bundle.AddLogicalImageTask_addingInterestingFiles());
+            addingInterestingFiles = true;
             addInterestingFiles(Paths.get(dest.toString(), resultsFilename), createVHD);
             progressMonitor.setProgressText(Bundle.AddLogicalImageTask_doneAddingInterestingFiles());
-            if (addMultipleImageTask != null) {
-                callback.done(addMultipleImageTask.getResult(), addMultipleImageTask.getErrorMessages(), addMultipleImageTask.getNewDataSources());
+            if (addMultipleImageTask != null && privateCallback != null) {
```

Comment:
If you move everything into one try-catch you won't need this null check

Reply:
I think "if (createVHD)" will also work.